src/main/java/ezvcard/util/JCardPrettyPrinter.java

```diff
@@ -0,0 +1,101 @@
+package ezvcard.util;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.*;
+import com.fasterxml.jackson.core.util.DefaultIndenter;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+
+import ezvcard.VCardVersion;
+import ezvcard.property.VCardProperty;
+
+public class JCardPrettyPrinter extends DefaultPrettyPrinter {
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * Alias for {@link DefaultIndenter#SYSTEM_LINEFEED_INSTANCE}
+	 */
+	public static final Indenter NEWLINE_INDENTER = DefaultIndenter.SYSTEM_LINEFEED_INSTANCE;
+	/**
+	 * Instance of {@link DefaultPrettyPrinter.FixedSpaceIndenter}
+	 */
+	public static final Indenter INLINE_INDENTER = new DefaultPrettyPrinter.FixedSpaceIndenter();
+
+	private Indenter propertyIndenter = INLINE_INDENTER;
+	private Indenter arrayIndenter = NEWLINE_INDENTER;
+	private Indenter objectIndenter = NEWLINE_INDENTER;
+
+	public JCardPrettyPrinter() {
+		super.indentArraysWith(arrayIndenter);
+		super.indentObjectsWith(objectIndenter);
+	}
+	
+	public JCardPrettyPrinter(JCardPrettyPrinter base) {
+		super(base);
+		this.propertyIndenter = base.propertyIndenter;
+		this.arrayIndenter = base.arrayIndenter;
+		this.objectIndenter = base.objectIndenter;
+		super.indentArraysWith(arrayIndenter);
+		super.indentObjectsWith(objectIndenter);
+	}
+	
+	@Override
+	public DefaultPrettyPrinter createInstance() {

```

Comment:
I think it would be better if the signature of this method returned a `JCardPrettyPrinter`.  This should not create a compilation error because `JCardPrettyPrinter` extends `DefaultPrettyPrinter`.